### PR TITLE
Fixed golint issue

### PR DESCRIPTION
**Pull request by kaustumbh7**
Fixes #2144 https://github.com/openebs/openebs/issues/2144

### CODE REVIEW

**Comment by sonasingh46** [comment]
@kaustumbh7 
Please paste issue link in PR description.
e.g
fixes <issue_link>

**Comment by kaustumbh7** [comment]
@sonasingh46 Just did.

File: pkg/install/v1alpha1/config.go [hunk 1]
```diff
@@ -16,10 +16,13 @@ limitations under the License.
 
 package v1alpha1
 
+// InstallConfig contains installation configuration specifications
```
Comment (sonasingh46) [inline pkg/install/v1alpha1/config.go]: I would add a period at the end of comment

File: pkg/install/v1alpha1/config.go [hunk 2]
```diff
@@ -16,10 +16,13 @@ limitations under the License.
...
 type InstallConfig struct {
 	Spec InstallConfigSpec `json:"spec"`
 }
 
+// InstallConfigSpec caters to installation and un-installation of
+// artifacts understood by installer
```
Comment (sonasingh46) [inline pkg/install/v1alpha1/config.go]: Here also, adding a period is what I would prefer.

**Pull request merged**